roles/etcd/tasks/tls.yml

```diff
@@ -1,5 +1,3 @@
----
```

Comment:
removing the `---` at the beggining of the file would not pass ansible-lint if it were enabled